modules/phase_field/src/auxkernels/EBSDReaderAvgDataAux.C

```diff
@@ -54,7 +56,9 @@ EBSDReaderAvgDataAux::precalculateValue()
 
   // get the data for the grain
   else
-    _value = (*_val)(_ebsd_reader.getAvgData(grain_id));
+    _value = _phase && _phase != libMesh::invalid_uint
```

Comment:
why the `_phase &&`? This will treat phase 0 differently than the rest. I thought we agreed that invalid_uint was supposed to be the magic value, not zero.